easybuild/tools/config.py

```diff
@@ -448,16 +450,6 @@ def get_log_filename(name, version, add_salt=False):
     return filepath
 
 
-def read_only_installdir():
-    """
-    Return whether installation dir should be fully read-only after installation.
-    """
```

Comment:
but this breaks the api?
Shouldn't you deprecate this function, and let it return the result of the config build option?